### DISCUSSION TITLE
hide pen ✏️ (toot) button

### DISCUSSION THREAD

**Discussion by yukulele** [NickColley/semaphore#43] [New features and ideas]
new toot (✏️) button over content can be distracting.

I suggest to add a "wellness" option to hide it (or move it on menu bar)

**Comment by yukulele** [comment]
https://github.com/nolanlawson/pinafore/issues/2288
https://github.com/nolanlawson/pinafore/issues/425

**Comment by NickColley** [comment]
How would you feel if we moved it to the bottom right on smaller screens? I think there's some work to simplify how we manipulate the header position rather than making it a user setting.
